### DISCUSSION TITLE
Required changes for K9Dozer blade

### DISCUSSION THREAD

**Discussion by nitroSkirata** [Courseplay/Courseplay_FS22#3089] [Q&A]
Hey,

I am looking for what needs to be modified to make the courseplay leveler height control work with the K9 blade from modhub.
https://www.farming-simulator.com/mod.php?lang=en&country=gb&mod_id=282265&title=fs2022

In an earlier issue it was mentioned the blade needs to be modified for CP to be able to use it, but the specific problems between CP and the blade were sadly not mentioned.

I'd really like to be able to modify the blade to work, but at least on a cursory glance nothing stands out to me

**Comment by Tensuko** [comment]
The simple answer is that it have to stick to Giants standards.
If that blade is not for 3 point it should be for wheelloader if that is not the case either then if have to be made like one or another.
